hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/evm/MirrorEvmTxProcessorTest.java

```diff
@@ -0,0 +1,244 @@
+
+    private final MirrorEvmAccount sender = new MirrorEvmAccount(Address.ALTBN128_ADD);
+    private final MirrorEvmAccount receiver = new MirrorEvmAccount(Address.ALTBN128_MUL);
+    private final Address receiverAddress = receiver.getAddress();
+    private final Instant consensusTime = Instant.now();
+    private MirrorEvmTxProcessor mirrorEvmTxProcessor;
+
+    @BeforeEach
+    void setup() {
+        setupGasCalculator();
+        var operationRegistry = new OperationRegistry();
+        MainnetEVMs.registerLondonOperations(operationRegistry, gasCalculator, BigInteger.ZERO);
+        operations.forEach(operationRegistry::put);
+        String EVM_VERSION_0_30 = "v0.30";
+        when(evmProperties.evmVersion()).thenReturn(EVM_VERSION_0_30);
+        var evm30 = new EVM(operationRegistry, gasCalculator, EvmConfiguration.DEFAULT);
+        String EVM_VERSION_0_32 = "v0.32";
+        Map<String, Provider<MessageCallProcessor>> mcps =
+                Map.of(
+                        EVM_VERSION_0_30,
+                        () -> new MessageCallProcessor(
+                                evm30, new PrecompileContractRegistry()),
+                        EVM_VERSION_0_32,
+                        () -> new MessageCallProcessor(
+                                evm30, new PrecompileContractRegistry()));
+        Map<String, Provider<ContractCreationProcessor>> ccps =
+                Map.of(
+                        EVM_VERSION_0_30,
+                        () -> new ContractCreationProcessor(
+                                gasCalculator, evm30, true, List.of(), 1),
+                        EVM_VERSION_0_32,
+                        () -> new ContractCreationProcessor(
+                                gasCalculator, evm30, true, List.of(), 1));
+
+        mirrorEvmTxProcessor =
+                new MirrorEvmTxProcessor(
+                        worldState,
+                        pricesAndFeesProvider,
+                        evmProperties,
+                        gasCalculator,
+                        mcps,
+                        ccps,
+                        blockMetaSource,
+                        hederaEvmContractAliases,
+                        hederaEvmEntityAccess);
+
+        MirrorOperationTracer hederaEvmOperationTracer = new MirrorOperationTracer();
+        mirrorEvmTxProcessor.setOperationTracer(hederaEvmOperationTracer);
+    }
+
+    @Test
+    void assertSuccessExecution() {
+        givenValidMockWithoutGetOrCreate();
+        given(evmProperties.fundingAccountAddress())
+                .willReturn(Address.ALTBN128_PAIRING);
+        given(hederaEvmContractAliases.resolveForEvm(receiverAddress)).willReturn(receiverAddress);
+
+        var result =
+                mirrorEvmTxProcessor.execute(
+                        sender, receiverAddress, 33_333L, 1234L, Bytes.EMPTY, consensusTime, true);
+        assertTrue(result.isSuccessful());
+        assertEquals(receiver.getAddress(), result.getRecipient().get());
```

Comment:
In general, we prefer chaining assertj assertions on mirror node.
```suggestion
        assertThat(result)
            .isNotNull()
            .returns(true, HederaEvmTransactionProcessingResult::isSuccessful)
            .returns(receiver.getAddress(), r -> r.getRecipient().get());
```